src/client/sandbox/event/timeout.js

```diff
@@ -1,86 +1,90 @@
+import SandboxBase from '../base';
```

Comment:
Since it's replaces both `setTimeout` and `setInterval` seems like it's better be named `TimersSandbox`